### ISSUE TITLE
JAWS Issues around Programmatically Setting Focus to Spans

### ISSUE THREAD

**Issue by msolf-GitHub** [FreedomScientific/VFO-standards-support#525]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://jsfiddle.net/msolf_jsfiddle/ekvg76so/35/show
> 2. Click on Case 1 or the Case 2 buttons to move the focus to elements on the screen
> 2a. For Case 1, notice JAWS does not read the text in JAWS 2021 (but does in JAWS 2020)
> 2b. For Case 2, click the down arrow to notice the focused area jumps (in JAWS 2020/2021 but not NVDA)

When the focus is set programmatically to a span (with a tabindex of -1), JAWS behaves unpredictably in the following cases;
**Case 1;**
When the focus is set to an instruction label, in JAWS 2021 it would sometimes not read the text.
**Case 2;**
When the focus is set to a prompt for a fieldset, clicking down arrow will result in JAWS jumping to another area of the screen.

### Expected result
> What did you expect to happen?

The following was the expected results for the cases;
**Case 1;**
     After drawing focus to the instruction label, the screen reader would read the show instructions label.
**Case 2;**
     After drawing focus to the prompt label, clicking down would read the radio buttons below.

### Actual result
> What actually did happen?

The following was the actual results for the cases;
**Case 1;**
     After drawing focus to the instruction label, the screen reader would not read the show instructions label in JAWS 2021 (JAWS 2020 does).
**Case 2;**
     After drawing focus to the prompt label, clicking down would cause the focus to jump to the top or the bottom of the screen (unlike other screen readers like NVDA).

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
An example of this was built in [JSFiddle](https://jsfiddle.net/msolf_jsfiddle/ekvg76so/35/) ([Website without code link](https://jsfiddle.net/msolf_jsfiddle/ekvg76so/35/show)).

**Case 1;**
1. In the website, click the Case 1 button to draw focus to the "Case 1:..." instruction label.
2. Note in JAWS 2020 it reads the label, but doesnt for JAWS 2021.

**Case 2;**
1. In the website, click the Case 2 button to draw focus to the "Case 2:..." prompt label.
2. Once the focus is drawn to it, click the down arrow to find the focus doesnt go to the radio buttons below the case 2 prompt label.
3. Note this was quickly written as a verification, you may have to click the "DoesNothing" button between clicks of Case 2.

#### JAWS version and build number
JAWS 2020 [Version 2020.2012.13 ILM]
JAWS 2021 [Version 2021.2102.34 ILM]

 #### Operating System and version
Windows 10 [Version 20H2]

#### Browser and version:
Google Chrome [Version 90.0.4430.93 (Official Build) (64-bit)]

**Comment by JAWS-test** [comment]
Related: https://github.com/FreedomScientific/VFO-standards-support/issues/508
